config/routes.rb

```diff
@@ -63,13 +63,14 @@
   namespace :merchant do
     get '/dashboard', to: 'dashboard#show'
     get '/items', to: 'items#index'
+    get '/orders/:id', to: 'orders#show'
```

Comment:
i feel like it should be ok to access the regular order show page, but this works